api/vudl.ini.dist

```diff
@@ -333,4 +333,18 @@ name = "Attribution-NonCommercial-NoDerivs"
 uri = "http://creativecommons.org/licenses/by-nc-nd/4.0/"
 [licenses.PUBLIC_DOMAIN]
 name = "CC0 1.0 Universal (Public Domain)"
-uri = "https://creativecommons.org/publicdomain/zero/1.0/"
\ No newline at end of file
+uri = "https://creativecommons.org/publicdomain/zero/1.0/"
+
+# Agent
+[agent]
+
```

Comment:
Can we put an example in here of how to set agent defaults? Even if it's commented out, it would probably be helpful for clarity.